app/pages/lab/workflow-create-form.cjsx

```diff
@@ -27,6 +27,7 @@ WorkflowCreateForm = createReactClass
     newWorkflow =
       id: workflowToClone?.id
       display_name: @refs.newDisplayName.value
+      primary_language: workflowToClone?.primary_language || @props.project.primary_language
```

Comment:
`workflowToClone` will always be undefined for a new workflow, and only the ID is used when copying a workflow. So you can remove it here.